store/extensions/G-Midi/extension.json

```
@@ -0,0 +1,33 @@
+{
+    "title": "G-Midi",
+    "description": "Play .midi files on 37-key pianos \n\nAutomatically plays piano notes from the uploaded MIDI file",
+    "authors": [
+      {
+        "name": "-Yıldırım",
+        "hotel": ".com.tr",
+        "username": "-Yıldırım",
+        "discord": "yildirimmmmmm",
+        "twitter": "HabboYildirim"
+      }
+    ],
+    "version": "1.0",
+    "categories": ["Fun", "Others"],
+    "source": "https://github.com/yildirimmmm/G-Midi",
+    "readme": "https://github.com/yildirimmmm/G-Midi#readme",
+    "stable": true,
+    "framework": {
+      "name": "G-Node",
+      "version": "0.2.16"
+    },
```

Your package.json lists 0.2.18
```suggestion
    "framework": {
      "name": "G-Node",
      "version": "0.2.18"
    },
```